hub/config/src/main/java/uk/gov/ida/hub/config/application/PrometheusClientService.java

```diff
@@ -12,7 +12,13 @@
 
 public class PrometheusClientService {
     public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY = "verify_config_certificate_expiry";
+    public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY_TIMESTAMP = "verify_config_certificate_expiry_timestamp";
     public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS = "verify_config_certificate_ocsp_success";
+    public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS_TIMESTAMP = "verify_config_certificate_ocsp_success_timestamp";
+    public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY_HELP = "Timestamp Of NotAfter Value Of X.509 Certificate";
+    public static final String VERIFY_CONFIG_CERTIFICATE_EXPIRY_TIMESTAMP_HELP = "Certificate Expiry Date Metric Timestamp";
+    public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS_HELP = "X.509 Certificate OCSP Revocation Status";
+    public static final String VERIFY_CONFIG_CERTIFICATE_OCSP_SUCCESS_TIMESTAMP_HELP = "X.509 Certificate OCSP Revocation Status Metric Timestamp";
```

Comment:
similarly, can the help text say something like "The timestamp of the last time we got a successful OCSP response"

Reply:
I have updated it to `X.509 Certificate OCSP Revocation Status Metric Last Updated (ms)`.